TddShop/TddShop.Cli/Order/Models/OrderModel.cs

```diff
@@ -10,5 +10,11 @@ public class OrderModel
     {
         public string CustomerUsername { get; set; }
         public ItemModel[] Items { get; set; }
+        public DateTime CreatedAt { get; set; }
```

Comment:
Hmm jeśli damy tutaj CreatedAt (CreatedOn w ogóle chyba powinno być) to wtedy ten test, gdzie mamy przekazywanego DateTime'a, żeby uniezależnić się od DateTime.Now przestaje mieć sens. Może tak samo tutaj przekażemy przez parametr datę złożenia zamówienia?

Reply:
`CreatedAt` vs `CreatedOn` - różnie widywałem :)
Jeśli test z przekazywanym DateTime też jest oparty o `OrderModel` to dla uproszczenia dorzućmy sobie parametr z tą datą do naszej testowanej metody.